images/Dockerfile.ubuntu.certified.ldapd

```diff
@@ -0,0 +1,10 @@
+FROM ansible/ubuntu-ssh
+MAINTAINER Harish Anand "https://github.com/harishanand95"
+
+RUN echo "TrustedUserCAKeys /etc/ssh/ca-user-certificate-key.pub" >> /etc/ssh/sshd_config
+
+EXPOSE 22
+EXPOSE 8389
```

Comment:
PORT 8389 is exposed for using apache directory studio. @pct960 IPtables in the server should block based on requirements.